iron-fit-behavior.html

```diff
@@ -359,27 +365,18 @@
 
       var position = this.__getPosition(this._localeHorizontalAlign, this.verticalAlign, size, positionRect, fitRect);
 
+      var minLeft = margin.left + (position.horizontalAlign === 'left' ? this.horizontalOffset : 0);
+      var minTop = margin.top + (position.verticalAlign === 'top' ? this.verticalOffset : 0);
```

Comment:
Shouldn't `fitRect.left` and `fitRect.top` be added to `minLeft` and `minTop` also?